Gordon360/Services/RecIM/ActivityService.cs

```diff
@@ -26,15 +27,15 @@ public IEnumerable<LookupViewModel> GetActivityLookup(string type)
             switch (type)
             {
                 case "status":
-                    return _context.ActivityStatus
+                    return _context.ActivityStatus.Where(query => query.ID != 0)
                                 .Select(s => new LookupViewModel
                                 {
                                     ID = s.ID,
                                     Description = s.Description
                                 })
                                 .AsEnumerable();
                 case "activity":
-                    return _context.ActivityType
+                    return _context.ActivityType.Where(query => query.ID != 0)
```

Comment:
The thing I mentioned revolves around actually just modifications and not the actual query itself. I could make a helper function but the issue is that the helper function wouldn't save any lines of code and most likely will add lines. HOWEVER, it would be good for centralized change if the deleted status changes ID